_sass/blocks/explore/_filters.scss

```diff
@@ -76,7 +76,17 @@
 // }
 
 .unit-selector {
+  display: inline-block;
```

Comment:
Properties should be ordered background, background-color, display, height, width